recipes/pyqode.python/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "pyqode.python" %}
+{% set version = "2.10.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e03855e70fef8606f378afa3ac41938eab601749bdfd00d922bb3fc7c4dccd81
+
+build:
+  number: 0
+  noarch: python
+  preserve_egg_dir: True
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - docutils
+    - jedi
+    - pep8
+    - pip
+    - pyflakes
+    - pyqode.core
+    - pyqode.qt
+    - python
+    - setuptools
+  run:
+    - docutils
+    - jedi
+    - pep8
+    - pyflakes
```

Comment:
are ``pep8`` and ``pyflakes`` necessary to run this package? Or just in test phase?